libres_util/src/res_log.c

```diff
@@ -16,14 +16,25 @@
    for more details. 
 */
 
+/*TODO:
+ * - Go over the current log-calls and make sure they use reasonable log-levels
+ * - Should we change log.h to use the normal semantics of log-levels, where higher means more important message?
```

Comment:
That must actually be done in libecl, I just used this list to remember the suggestion. I will make a task on it for libecl and do it. I will then also change the numeric values of the different levels to correspond to Pythons, so we have the same semantic as Python.